src/renate/benchmark/datasets/wild_time_data.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from pathlib import Path
+from typing import Any, Dict, Optional, Union
+
+from transformers import PreTrainedTokenizer
+from wild_time_data import load_dataset
+from wild_time_data.core import available_time_steps, dataset_classes
+
+from renate import defaults
+from renate.data.data_module import RenateDataModule
+from renate.utils.file import download_folder_from_s3
+
+
+class WildTimeDataModule(RenateDataModule):
+    """Data module wrapping torchvision datasets.
+
+    Args:
+        data_path: the path to the folder containing the dataset files.
+        dataset_name: Name of the wild time dataset.
+        src_bucket: the name of the s3 bucket. If not provided, downloads the data from original
+            source.
+        src_object_name: the folder path in the s3 bucket.
+        time_step: Time slice to be loaded.
+        tokenizer: Tokenizer to apply to the dataset. See https://huggingface.co/docs/tokenizers/
+            for more information on tokenizers.
+        tokenizer_kwargs: Keyword arguments passed when calling the tokenizer's ``__call__``
```

Comment:
It would be great to report which one was used in the paper describing the dataset

Reply:
The use DistilBERT but not really anything specific for the dataset.